openedx/core/djangoapps/api_admin/management/commands/create_api_access_request.py

```diff
@@ -0,0 +1,130 @@
+""" Management command to create an ApiAccessRequest for given users """
+import logging
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+
+from openedx.core.djangoapps.api_admin.models import (
+    ApiAccessConfig,
+    ApiAccessRequest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Create an ApiAccessRequest for the given user
+
+    Example usage:
+        $ ./manage.py lms create_api_request <username> --create-config
+    """
+
+    help = 'Create an ApiAccessRequest for the given user'
+    DEFAULT_STATUS = ApiAccessRequest.PENDING
```

Comment:
our story has this defaulting to APPROVED

Reply:
good catch